File: 461.cpp
#include<iostream>

using namespace std;

class Solution {
public:
    int hammingDistance(int x, int y) {
        int x_xor_y = x^y;
        int cnt = 0;
        
        for(int i=0;i<32;++i){
            if(1&(x_xor_y >> i)){
                cnt++;
            }
        }
        
        return cnt;
    }
};

int main(){
    
}

File: Python/461.py
class Solution(object):
    def hammingDistance(self, x, y):
        """
        :type x: int
        :type y: int
        :rtype: int
        """
        z = x^y
        count = 0
        while z>0:
            if z%2:
                count+=1
            z /= 2
        return count
